launch_testing/launch_testing_examples/launch_testing_examples/set_param_launch_test.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2021 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import launch
+import launch.actions
+import launch_ros.actions
+import launch_testing.actions
+import launch_testing.markers
+import pytest
+from rcl_interfaces.srv import SetParameters
+import rclpy
+from rclpy.node import Node
+
+
+@pytest.mark.launch_test
+@launch_testing.markers.keep_alive
+def generate_test_description():
+    return launch.LaunchDescription([
+        launch_ros.actions.Node(
+            executable='parameter_blackboard',
+            package='demo_nodes_cpp',
+            name='demo_node_1'
+        ),
+        launch_testing.actions.ReadyToTest()
+    ])
+
+
+class TestFixture(unittest.TestCase):
+
+    def test_set_parameter(self, proc_output):
+        rclpy.init()
+        node = MakeTestNode('test_node')
+        response = node.set_parameter(state=True)
```

Comment:
Same comment here, consider,
```suggestion
        node = Node('test_node')
        response = set_parameter(node, value=True)
```

Also, the name `value` for the argument is more meaningful to me in this context.